File: backend/backend/Entities/CommissionForm.cs
using System.ComponentModel.DataAnnotations;

namespace backend.Entities
{
    public class CommissionForm
    {
        [Key]
        public int CommissionFormID { get; set; }
        public int CommissionID { get; set; } // PK
        public int ReceiverID { get; set; } // FK
        public int RequestorID { get; set; } // FK
        public string Description { get; set; }
        public Boolean?  Accept  { get; set; }
        public int? Progress { get; set; }
    }
}

File: frontend/src/Interfaces/CommissionForm.ts
export interface ICommissionForm {
    commissionFormID: number|0,
    commissionID: number|0,
    receiverID: string|number,
    requestorID: string|number,
    description: string|""
    accept: boolean|null,
    progress: number|0,
}
export interface IExtraCommissionForm{
    commissionFormID: number|0,
    commissionID: number|0,
    receiverID: string|number,
    requestorID: string|number,
    description: string|""
    accept: boolean|null,
    progress: number|0,
    requestorEmail: string,
    requestorPhone: string,
    requestorUserName: string,
}

export interface ICommissionID {
    commissionID: string|number,
   
}